src/dmd/mtype.d

```diff
@@ -214,6 +214,16 @@ char* MODtoChars(MOD mod) nothrow
     return buf.extractString();
 }
 
+/// Ditto
+const(char)[] MODtoString(MOD mod) nothrow
+{
+    OutBuffer buf;
+    buf.reserve(16);
+    MODtoBuffer(&buf, mod);
+    size_t len = buf.offset;
+    return buf.extractData()[0 .. len];
```

Comment:
This implementation should be swapped with the implementation of ModtoBuffer, then there will be no need to make the copy.